tests/ts/optional_scalars/scalar-stuff.ts

```diff
@@ -0,0 +1,423 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
This should probably sit in `tests/optional_scalars/scalar-stuff.ts`, to be analogous to other languages.